docker-compose.x86.darknet.gpu.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.4'
+
+services:
+  ml_api:
+    image: thespaghettydetective/ml_api_x86_64_darknet_gpu
+    build:
+      dockerfile: Dockerfile.x86_64
+      target: ml_api_darknet_gpu
+      context: ml_api
+    # enables GPU access for container
+    deploy:
+      resources:
+        reservations:
```

Comment:
I don't think this is necessary. Our current version doesn't force a resource reservation. If users wants to do it, they can do that via docker-compose.override.yaml. I don't think we want to force this upon users by defau.t

Reply:
I can remove that. Just need to be sure, that we have a documentation how to run this docker-compose image on CUDA as it may not run without a reservation.